### PR TITLE
feat(IdentityCenter): add Identity Center group resource

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
   add Identity Center group resource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
   add Identity Center group resource
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
go test .\huaweicloud\services\acceptance\identitycenter\ -v -run TestAccIdentityCenterGroup_basic
=== RUN   TestAccIdentityCenterGroup_basic
=== PAUSE TestAccIdentityCenterGroup_basic
=== CONT  TestAccIdentityCenterGroup_basic
--- PASS: TestAccIdentityCenterGroup_basic (11.13s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/identitycenter    16.240s
```

### CODE REVIEW

File: docs/incubating/identitycenter_group.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* `created_at` - Indicates the creation time.
+
+* `updated_at` - Indicates the last update time.
+
+## Import
+
+The Identity Center group can be imported using the `identity_store_id` and `id` separated by a slash, e.g.
+
+```bash
+$ terraform import huaweicloud_identitycenter_group.test <identity_store_id>/<id>
```
Comment (niuzhenguo) [inline docs/incubating/identitycenter_group.md:52]: `identity_store_id` seems unnecessary here, as when importing, the resource defination with the required parameter already there, you can just use the current Read logic.

Comment (houpeng80) [reply docs/incubating/identitycenter_group.md:52]: when import a resource, the param that in URL path should be imported at the same time

Comment (niuzhenguo) [reply docs/incubating/identitycenter_group.md:52]: yes, d.Get in Read when importing will get empty value as it't not in state yet.

File: docs/incubating/identitycenter_group.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Manages an Identity Center group resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_identitycenter_instance" "system" {}
+
+resource "huaweicloud_identitycenter_group" "test"{
+  identity_store_id = data.huaweicloud_identitycenter_instance.test.identity_store_id
```
Comment (ShiChangkuo) [inline docs/incubating/identitycenter_group.md]:
```suggestion
  identity_store_id = data.huaweicloud_identitycenter_instance.system.identity_store_id
```

File: huaweicloud/services/identitycenter/resource_huaweicloud_identitycenter_group.go [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+	updateIdentityCenterGroupPath := updateIdentityCenterGroupClient.Endpoint + updateIdentityCenterGroupHttpUrl
+	updateIdentityCenterGroupPath = strings.ReplaceAll(updateIdentityCenterGroupPath, "{identity_store_id}",
+		fmt.Sprintf("%v", d.Get("identity_store_id")))
+	updateIdentityCenterGroupPath = strings.ReplaceAll(updateIdentityCenterGroupPath, "{group_id}", d.Id())
+
+	updateIdentityCenterGroupOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+	updateIdentityCenterGroupOpt.JSONBody = utils.RemoveNil(buildUpdateIdentityCenterGroupBodyParams(d))
```
Comment (ShiChangkuo) [inline huaweicloud/services/identitycenter/resource_huaweicloud_identitycenter_group.go:195]: if there is another updateable parameter,  buildUpdateIdentityCenterGroupBodyParams maybe return `map[string]interface{}{"operations": []}` when description is not changed and the API will fail.

so I think it's better to add d.HasChange in the outside of `buildUpdateIdentityCenterGroupBodyParams `

File: docs/incubating/identitycenter_group.md (hunk 2 above)
Comment (houpeng80) [reply docs/incubating/identitycenter_group.md]: done

File: huaweicloud/services/identitycenter/resource_huaweicloud_identitycenter_group.go (hunk 3 above)
Comment (houpeng80) [reply huaweicloud/services/identitycenter/resource_huaweicloud_identitycenter_group.go:195]: done

**Pull request merged**